src/crypto/entropy.cpp

```diff
@@ -16,6 +16,6 @@ namespace crypto
       return intel_drng_ptr;
     }
 
-    return std::make_shared<Entropy_mbedTLS>();
+    return std::make_shared<Entropy_OpenSSL>();
```

Comment:
Would what be possible?

Reply:
Sorry, rewording. MBedTLS would pass the entropy source to the configuration, but I'm not doing that at the moment in OpenSSL because I couldn't find a way to do that in the same way mbed does. so the current openssl uses the standard entropy source, which I guess it's good enough by reading the OpenSSL docs.

Reply:
Yes, that's right, the default initialization is fine for OpenSSL. We still have an entropy.h for random number generation in other places though (e.g. ledger secrets) and that is fixed to using the Intel DRNG where supported.